app-text/sigil/files/sigil-0.9.13-fix-install.patch

```diff
@@ -0,0 +1,62 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index e6d741e..cfa0ef7 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -33,6 +33,8 @@ set( SIGIL_REVISION_VERSION 13 )
+ set( SIGIL_FULL_VERSION ${SIGIL_MAJOR_VERSION}.${SIGIL_MINOR_VERSION}.${SIGIL_REVISION_VERSION} )
+ 
+ if( UNIX AND NOT APPLE )
++   include(GNUInstallDirs)
++
+    # Make sure the correct rpath is set for the sigil executable
+    # on Linux -- for both the build and the installed versions.
+    # Needed to make sure the installed shared hunspell and gumbo
+@@ -45,7 +47,7 @@ if( UNIX AND NOT APPLE )
+    # (but later on when installing)
+    SET(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+ 
+-   SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib/sigil")
++   SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_FULL_LIBDIR}/sigil")
```

Comment:
I'm sorry but now that I see the patch, I don't think libdir is really necessary here. As long as it installs only big 'sigil' directory, /usr/lib is preferable as it keeps the path constant.

Reply:
Done.
Now the patch can be removed.